Makefile

```diff
@@ -90,6 +90,27 @@ install: go.sum
 	go install -mod=readonly $(BUILD_FLAGS) ./cmd/gaiad
 	go install -mod=readonly $(BUILD_FLAGS) ./cmd/gaiacli
 
+
+########################################
+### Documentation
+
+build-docs:
+	@cd docs && \
+	while read p; do \
+		(git checkout $${p} && npm install && VUEPRESS_BASE="/$${p}/" npm run build) ; \
+		mkdir -p ~/output/$${p} ; \
+		cp -r .vuepress/dist/* ~/output/$${p}/ ; \
+		cp ~/output/$${p}/index.html ~/output ; \
+	done < versions ;
+
+sync-docs:
+	cd ~/output && \
```

Comment:
This is really only run in the CI process. Does it need to be in the makefile? Most devs won't have or even use `aws`.

Reply:
I don't want to make a separate makefile just to deploy the docs. Nor do I want to add it to the Circle config which is huge as is and could use more trimming.